### PR TITLE
feat(trading): rewards page

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

Partially addresses #4532 

Does not contain
- Activity streak
- Tier cards

# Description ℹ️

Adds rewards tab to the portfolio page

# Demo 📺

Connected:

Screenshot 2023-11-14 at 11 45 21

Disconnected:

Screenshot 2023-11-14 at 11 45 55

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: apps/trading/components/rewards-container/rewards-container.tsx [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        </tr>
+        {pubKey && (
+          <>
+            <tr>
+              <CardTableTH>{t('Vesting multiplier')}</CardTableTH>
+              <CardTableTD>0%</CardTableTD>
+            </tr>
+            <tr>
+              <CardTableTH>{t('Available to withdraw next epoch')}</CardTableTH>
+              <CardTableTD>0</CardTableTD>
```
Comment (asiaznik) [inline apps/trading/components/rewards-container/rewards-container.tsx:309]: question: What if there's something in a `vesting` account. Shouldn't we cover that here?

Comment (asiaznik) [reply apps/trading/components/rewards-container/rewards-container.tsx:309]: or is it impossible now to have anything in `vesting`?

**Comment by bwallacee** [comment]
@mattrussell36 should i be able to go down to -1 epoch?
image


Also i wonder if it would be useful to have a button or something to put the "to" epoch to current?

**Comment by mattrussell36** [comment]
> @mattrussell36 should i be able to go down to -1 epoch? image
> 
> Also i wonder if it would be useful to have a button or something to put the "to" epoch to current?

No, this should never bee negative, @bwallacee This is fixed.

File: apps/trading/components/rewards-container/rewards-container.tsx (hunk 1 above)
Comment (mattrussell36) [reply apps/trading/components/rewards-container/rewards-container.tsx]: When I picked this one up it was agreed to skip over the vesting card. But it now looks like the required apis are in, so I've updated it

**Comment by bwallacee** [comment]
LGTM, but do we need specs added for this?

**Review by bwallacee — APPROVED** [review]

File: apps/trading/components/rewards-container/rewards-container.tsx (hunk 1 above)
Comment (mattrussell36) [reply apps/trading/components/rewards-container/rewards-container.tsx]: @bwallacee  @asiaznik  looks like I was using the wrong value here, so back to draft for this one

Comment (mattrussell36) [reply apps/trading/components/rewards-container/rewards-container.tsx]: Please take another look

**Comment by mattrussell36** [comment]
> @mattrussell36 retested and it LGTM, but do we need specs added for this?

@bwallacee I've added some specs

**Review by MadalinaRaicu — APPROVED** [review]

**Pull request merged**